src/views/catalog/customize/cd.ts

```diff
@@ -11,24 +12,36 @@ import {
   replaceTemplateVM,
 } from '@kubevirt-utils/resources/template';
 
+import { removeDockerPrefix } from './components/CustomizeSource/utils';
 import { INSTALLATION_CDROM_DISK, INSTALLATION_CDROM_NAME } from './constants';
 
 export const addInstallationCDRom = (
   virtualMachine: V1VirtualMachine,
-  cdSource: V1beta1DataVolumeSpec,
+  cdSource: V1beta1DataVolumeSpec | V1ContainerDiskSource,
 ): V1VirtualMachine => {
   let cdVolume: V1Volume = undefined;
   let cdDataVolumeTemplate: V1DataVolumeTemplateSpec = undefined;
   const dataVolumeName = `${virtualMachine?.metadata?.name}-${INSTALLATION_CDROM_NAME}`;
 
-  if (cdSource?.source?.registry) {
+  if ((cdSource as V1ContainerDiskSource)?.image) {
     cdVolume = {
       name: INSTALLATION_CDROM_NAME,
       containerDisk: {
-        image: cdSource.source?.registry.url,
+        image: removeDockerPrefix((cdSource as V1ContainerDiskSource).image),
```

Comment:
let's have this `removeDockerPrefix` in case the user types the prefix anyways.
Here the prefix will cause an error in the VM start.